Statistics.cs

```diff
@@ -28,10 +31,9 @@ public Statistics(string fileName, DataModeler.SupportedFileTypes fileType)
         }
 
         // City Methods
-
         public bool IsValidCity(string cityName)
         {
-            return CityCatalogue.ContainsKey(cityName.ToLower());
+            return GetSpecificCity(cityName)!=null?true:false;
```

Comment:
We might want to remove this and just do that check wherever we would call IsValidCity as some methods require the valid city object (Comparing populations)

Reply:
I agree I think this method can probably just be switched out for GetSpecificCity. We can change the name of it if that makes it make more sense  but a function that just calls another function is a bit unnecessary. If we do keep it, it can be simplified to `return GetSpecificCity(cityName) != null;`